minio/1.11/README.md

```diff
@@ -0,0 +1,160 @@
+# Install and use Minio on DC/OS
+
+[Minio](https://minio.io) is an object storage server released under Apache License v2.0. It is compatible with Amazon S3 cloud storage service. It is best suited for storing unstructured data such as photos, videos, log files, backups and container / VM images. Size of an object can range from a few KBs to a maximum of 5TB.
+
+Minio server is light enough to be bundled with the application stack, similar to NodeJS, Redis and MySQL.
+
+This installation uses the single node version of Minio (Minio FS), checkout the [Minio docs](https://docs.minio.io) for more details.
+
+The instructions below use a pinned hostname constraint to ensure the application is always restarted on the same host by Marathon. This allows it to get back to its data but means that you could lose data if that agent goes down.
+
+- Estimated time for completion: less than 5 minutes.
+- Target audience:
+    - Operators
+    - Application admins
+    - Developers
+    - Devops Engineers
+- Scope: Learn to install Minio on DC/OS and learn to use it using Minio command line tool `mc`.
+
+**Table of Contents**:
+
+- [Prerequisites](#prerequisites)
+- [Minio Installation using GUI](#minio-installation-using-gui)
+- [Customize Minio using GUI](#customize-minio-using-gui)
+- [Minio Installation using CLI](#minio-installation-using-cli)
+- [Access Minio Browser](#access-minio-browser)
+- [Install Minio Client](#install-minio-client)
+- [Configure Minio Client](#configure-minio-client)
+- [Uninstall Minio using CLI](#uninstall-minio-using-cli)
+- [Explore Further](#explore-further)
+
+## Prerequisites
+
+- DC/OS 1.11 or later
+- [Marathon-LB](https://docs.mesosphere.com/services/marathon-lb/) must be installed and running
+- Identify [IP of the public agent](https://docs.mesosphere.com/1.11/administering-clusters/locate-public-agent/) where Marathon-LB or an available hostname configured to point to the public agent(s) where Marathon-LB is running.
+
+## Minio installation using GUI
+
+- Login to DC/OS admin, click on Catalog on the left menu bar, search `minio` and select the package with following logo.
+
+![Search & Install Minio](img/search-install.png)
+
+- Click on the `Review & Run` button for the single-click default installation. You can reach your Minio server at `host:9000` where `host` is IP address or hostname of public-agent where Marathon-LB is installed.
+
+![Single-click Install](img/minio-package-install-oneclick.png)
+
+- This installs Minio server instance with factory defaults
+
+- Go to `Services` from left menu bar and verify, minio service status should show `Running` or wait for the status update.
+
+![Minio Service](img/minio-service.png)
+
+- Once Minio has been deployed, navigate to the IP/hostname of Public Agent where Marathon-LB is installed and the port you specified. To customize deployed minio server, continue reading or go to [Access Minio Browser](#access-minio-browser) section.
```

Comment:
Since this is the one click install, they wouldn't have specified a port, it's just the default of 9000 so you can tell them to use that.